addons/trenches/functions/fnc_placeTrench.sqf

```diff
@@ -68,29 +68,35 @@ GVAR(digPFH) = [{
     // Stick the trench to the ground
     _basePos set [2, getTerrainHeightASL _basePos];
     private _minzoffset = 0;
-    for [{private _ix = -_dx/2},{_ix <= _dx/2},{_ix = _ix + _dx/3}] do {
-        for [{private _iy = -_dy/2},{_iy <= _dy/2},{_iy = _iy + _dy/3}] do {
+    private _ix = 0;
+    private _iy = 0;
+    for [{_ix = -_dx/2},{_ix <= _dx/2},{_ix = _ix + _dx/3}] do {
+        for [{_iy = -_dy/2},{_iy <= _dy/2},{_iy = _iy + _dy/3}] do {
```

Comment:
We never use this for loop in ACE. It is very slow compared to the for-from-to-do syntax. And it is also ugly af.

Reply:
That is the way the trenches work in ace. I do not have a better way.
https://github.com/acemod/ACE3/blob/master/addons/trenches/functions/fnc_placeTrench.sqf#L71

Reply:
changed in d235d08 to the for-from-to-do syntax.